src/modules/powerrename/lib/PowerRenameManager.cpp

```diff
@@ -1121,8 +1121,9 @@ DWORD WINAPI CPowerRenameManager::s_regexWorkerThread(_In_ void* pv)
                         std::wstring newNameToUseWstr{ newNameToUse };
                         PWSTR path = nullptr;
                         spItem->GetPath(&path);
+
                         // Following characters cannot be used for file names.
-                        // Ref https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#naming-conventions
+                        // Ref https://docs.microsoft.com/windows/win32/fileio/naming-a-file#naming-conventions
                         if (newNameToUseWstr.contains('<') ||
```

Comment:
I know it's nitpicking, but isn't this also doable with a Regular expression? And is `\\` a literal double backslash or an escaped one? In other words: is a _single_ backslash allowed?

Reply:
>I know it's nitpicking, but isn't this also doable with a Regular expression?

Yeah, it is. I started with one char and then kept adding them. It doesn't make much difference anyway imho

> And is \\ a literal double backslash or an escaped one? In other words: is a single backslash allowed?

Escaped. '\' is not allowed in any way

Reply:
Mind if I ask for a double-check? I'm thinking it's in quatation marks, so why does it need to be escaped? A quick test here & now gives me this:
1. use a file @ desktop\\`computers.xlsx`
2. search `com` and replace `com\`
3. klick **Apply**
4. Windows show messagebox "C:\users\Jay\desktop does not exist, want to create the folder?"
   ![image](https://user-images.githubusercontent.com/65828559/185114849-e64660db-3ca8-4fe8-b4b5-d98a383e6dfc.png)

5. I click NO (to be on the safe side) --> file is not changed
6. PowerRename GUI/data _does_ change
   ![image](https://user-images.githubusercontent.com/65828559/185114978-fa586d4e-1cdf-412c-9929-65835d656a86.png)

Reply:
yes, this is how it is now, and this PR will fix this issue by making \ invalid char (which it is by official docs). PowerRename can't reflect FileExplorer's rename result. Same issue happens when e.g. you rename a file where there is already a file with the same name in the same dir. File explorer will append (<number>) to your file, and PowerRename can't figure that out. This is yet to be solved (to add a check for existing files with the same name in the same dir before renaming)